packages/genesys-spark-components/src/components/stable/gux-tooltip-title/gux-tooltip-title.tsx

```diff
@@ -119,6 +120,7 @@ export class GuxTooltipTitle {
         .querySelector('slot')
         .assignedNodes()
         .map(node => node.textContent)
+        .map(node => node.replace('\n', ''))
```

Comment:
added because when the text is slotted in a span and the span is on a middle line, the mapping of textContent returns something like `['\n       ','Text','\n     ']`, which then breaks the tooltip

Reply:
Would it be simpler to just use `node.textContent.trim()` to remove all preceding & trailing whitespace?

Reply:
Yeah looks like that solves it without the extra map too.

I found another issue where it's not updating the tooltip if there is no resize and a wrapping element is slotted instead of just the text, because there doesn't seem to be a way to check if child nodes of a slot have been changed..

Reply:
> I found another issue where it's not updating the tooltip if there is no resize and a wrapping element is slotted instead of just the text, because there doesn't seem to be a way to check if child nodes of a slot have been changed..

I'm ok with this limitation. I don't think we need to support complex structure inside most of these contexts.

Reply:
For the original issue, there was also an item around the screenreader text not updating - do you plan to address that in this PR or a separate one? 

I'd also wonder if it's possible to write an E2E test for this that would render a badge with content that fits, verify there's no truncation, update the conent, and verify it truncates.

Reply:
So I checked and maybe because of my fix or maybe because of something else, but it only happens when it is text inside a span that is slotted because we have no way to trigger a rerender when a child of the slotted element changes. The slotted element itself has to be replaced in the dom to trigger an onslotchange